_style/method-invocation.md

```diff
@@ -92,35 +92,28 @@ gray area is short, operator-like methods like `max`, especially if commutative:
     // fairly common
     a max b
 
-Symbolic methods which take more than one parameter (they do exist!)
-may still be invoked using infix notation, delimited by spaces:
+Symbolic methods which take more than one parameter are discouraged.
```

Comment:
Good to remove exclamations in parenthesis - they are paradoxical (shouting with a  hushed voice!).